Pages/Shared/_Layout.cshtml

```diff
@@ -41,24 +41,30 @@
         </div>
         <div class="header_bottom">
             <div class="container-fluid">
-                <nav class="navbar navbar-expand-lg custom_nav-container">
+                <nav class="navbar navbar-expand-lg custom_nav-container navbar-expand-sm">
                     <div>
                         <a class="navbar-brand" asp-page="Index">
                         <img src="~/images/logo.png.png" height="70px" />
                     </a>
                     </div>
                     <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent" aria-controls="navbarSupportedContent" aria-expanded="false" aria-label="Toggle navigation">
-                        <span class=""></span>
+                        <span class="navbar-toggler-icon"></span>
                     </button>
                     <form class="search-form" method="get">
                         <input class="search-input" type="text" name="searchTerm" placeholder="Search..." />
                         <button class="search-button" type="submit">Search</button>
                     </form>
                     <div class="collapse navbar-collapse ml-auto" id="navbarSupportedContent">
-                        <ul class="navbar-nav  ">
+                        <ul class="navbar-nav">
                             <li class="nav-item active">
                                 <a class="nav-link" asp-page="Index">Home <span class="sr-only">(current)</span></a>
                             </li>
+                            <li class="nav-item active">
+                                <a class="nav-link" asp-page="About"> About <span class="sr-only">(current)</span></a>
+                            </li>
+                            <li class="nav-item active">
+                                <a class="nav-link" asp-page="Contact"> Contact <span class="sr-only">(current)</span></a>
+                            </li>
```

Comment:
-Are there pages for these links?  Avoid adding unimplemented features, as unused code is clutter, which is technical debt.
- Do you really need these pages?  Avoid adding additional pages, which aren't adding much value, and will distract the user.